sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -58,12 +58,18 @@ az cognitiveservices account keys list --resource-group <your-resource-group-nam
 Once you have the value for the API key and Region, create an `TranslatorCredential`. This will allow you to
 update the API key without creating a new client.
 
-With the value of the endpoint, `TranslatorCredential` and a `Region`, you can create the [TextTranslationClient][translator_client_class]:
+With the value of the `endpoint`, `credential` and a `region`, you can create the [TextTranslationClient][client_sample]:
 
-```Python
-text_translator = TextTranslationClient(credential = TranslatorCredential("<apiKey>", "<apiRegion>"));
+<!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
+
+```python
+ 
+   credential = TranslatorCredential(apikey, region)
+   text_translator = TextTranslationClient(credential)
```

Comment:
This example still seems to be incorrect, the endpoint is missing here

Reply:
I have run the snippet update tool and pushed the README.md files.  Corrected both problems.